oereb_client/views/index.py

```diff
@@ -166,6 +169,7 @@ def get_config(self):
             'view': self.config_.get('view', {}),
             'base_layer': self.config_.get('base_layer', {}),
             'availability': self.config_.get('availability', {}),
+            'mask_surrounding': self.config_.get('mask_surrounding', {}),
```

Comment:
You should remove the default value "{}". This way you could easily check if a mask layer is defined in the javascript code (mask_surrounding !== null).

Reply:
https://github.com/openoereb/oereb_client/blob/c131dd2015af83bba9c51bd382c9b4db7b55e9d9/oereb_client/views/index.py#L158-L164

Reply:
https://github.com/openoereb/oereb_client/blob/c131dd2015af83bba9c51bd382c9b4db7b55e9d9/oereb_client/views/index.py#L158-L164

Reply:
However, I also removed the default value "{}".